app/services/return-requirements/submit-remove.service.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+/**
+ * Manages removing the return requirement from the session when remove is confirmed
+ * @module SubmitRemoveService
+ */
+
+const SessionModel = require('../../models/session.model.js')
+
+/**
+ * Manages deleting the selected requirement in the session when remove is confirmed
+ *
+ * The return requirements session data is deleted when a user confirms via the cancellation button and the requirement
+ * is deleted from the session in the database.
+ *
+ * @param {string} sessionId - The UUID for the return requirement setup session record
+ *
+ * @returns {Promise} the promise returned is not intended to resolve to any particular value
+ */
+async function go (sessionId, requirementIndex) {
+  const session = await SessionModel.query().findById(sessionId)
+
+  return await _removeRequirementFromSession(session, requirementIndex)
```

Comment:
As we are awaiting this in the controller we don't need to call `await` here.

```suggestion
  return _removeRequirementFromSession(session, requirementIndex)
```